.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,3 +15,19 @@ List the steps needed to make sure this thing works
 - [ ] **Verify** the thing does not do what it should not
 - [ ] **Document** the thing and how it works ([Example](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/post/multi/gather/aws_keys.md))
 
+If you are opening a PR for a new module that exploits a **specific** piece of hardware or requires a **complex or hard-to-find** testing environment, we recommend that you send us a demo of your module executing correctly. Seeing your module in action will help us review your use case faster!
+
+
+Specific Hardware Examples:
+* Enterprise Switches (PR 13303)
+* IP Cameras (PR 13170)
+
+
+Complex Testing Enviroment Examples:
+* Multiple testing scenarios (PR 12096)
+
+
```

Comment:
I wonder if `(PR 1234)` may not be obvious to first time contributors as to what it means.

Maybe we could change this to something like:

```suggestion

Examples:

- Hardware - Switches, Routers, Cameras, etc.
- Software - Expensive software, not being available on Docker, etc.

```

Or maybe nuke it entirely? The paragraph above is pretty informational in its existing form 🤔

Reply:
I would say changing it to the second format and then adding some links to some examples in the form of (ex PR 12096[*insert link here*] ) would probably be the best way to do this imho.